clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -882,11 +882,8 @@ void tools::addOpenMPRuntimeLibraryPath(const ToolChain &TC,
 
 void tools::addArchSpecificRPath(const ToolChain &TC, const ArgList &Args,
                                  ArgStringList &CmdArgs) {
-  // Enable -frtlib-add-rpath by default for the case of VE.
-  const bool IsVE = TC.getTriple().isVE();
```

Comment:
thx for removing the special case!

(I'd have suggested not having it in the first place if i had seen https://reviews.llvm.org/D79411 )